packages/amplify_core/lib/src/types/api/graphql/graphql_request.dart

```diff
@@ -18,7 +18,7 @@ import 'package:amplify_core/amplify_core.dart';
 /// A GraphQL request with a few extra properties used to decode the response or use the correct API if the backend has multiple.
 abstract class GraphQLRequest<T>
     with AWSSerializable<Map<String, Object?>>, AWSDebuggable {
-  GraphQLRequest._({
+  GraphQLRequest({
```

Comment:
Yeah this is now an abstract class so it cannot be directly instantiated. This allows other parts of the codebase to subclass it while forcing users to choose one of the factory methods, like `raw` which returns a `GraphQLRequest<Map<String, Object?>>` instead of a `GraphQLRequest<dynamic>` like before.